templates/index.html

```diff
@@ -50,19 +50,54 @@ <h3 class="p-muted-heading">Official snaps from major publishers</h3>
   <div class="row">
     <div class="p-fluid-grid--centered">
       <span class="p-logo-image p-fluid-grid__item--small u-align--center">
-        <img src="https://assets.ubuntu.com/v1/aae93857-logo-mozilla--snapcraft-homepage.svg" alt="Mozilla" />
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/aae93857-logo-mozilla--snapcraft-homepage.svg",
+            alt="Mozilla",
+            width="146",
+            height="145"
+          )|safe
```

Comment:
Probably not something for this PR but enhancement to `image` itself, but flask suggests using `Markup` class, to automatically return markup that doesn't need to be filtered with `safe`: http://flask.pocoo.org/docs/1.0/api/#flask.Markup